.golangci.yml

```diff
@@ -88,5 +89,4 @@ linters-settings:
 
 run:
   timeout: 10m
-  concurrency: 4
-  skip-dirs-use-default: false
```

Comment:
lint error that this is deprecated and to use`issues.exclude-dirs-use-default` instead.